packages/graph-explorer/src/modules/CreateConnection/CreateConnection.tsx

```diff
@@ -34,7 +34,8 @@ type ConnectionForm = {
 };
 
 const CONNECTIONS_OP = [
-  { label: "PG (Property Graph)", value: "gremlin" },
+  { label: "PG (Property Graph) - Gremlin", value: "gremlin" },
+  { label: "PG (Property Graph) - OpenCypher", value: "openCypher" },
   { label: "RDF (Resource Description Framework)", value: "sparql" },
```

Comment:
Could we add the term - SPARQL for consistency please;
"RDF (Resource Description Framework) - SPARQL"

Reply:
Done